specification/storage/resource-manager/readme.python.md

```diff
@@ -33,6 +34,16 @@ output-folder: $(python-sdks-folder)/storage/azure-mgmt-storage/azure/mgmt/stora
 perform-load: false
 ```
 
+### Tag: package-2025-01 and python
+
+These settings apply only when `--tag=package-2025-01 --python` is specified on the command line.
+Please also specify `--python-sdks-folder=<path to the root directory of your azure-sdk-for-python clone>`.
+
+``` yaml $(tag) == 'package-2025-01'
+namespace: azure.mgmt.storage.v2025_01_01
+output-folder: $(python-sdks-folder)/storage/azure-mgmt-storage/azure/mgmt/storage/v2025_01_01
+```
+
 ### Tag: package-2024-01 and python
 
 These settings apply only when `--tag=package-2024-01 --python` is specified on the command line.
```

Comment:
Same with https://github.com/Azure/azure-rest-api-specs/pull/33529/files#r2074521282, you can remove `### Tag: package-2024-01 and python` part.